domain-management/src/main/java/org/jboss/as/domain/management/security/SecurityRealmAddHandler.java

```diff
@@ -674,11 +674,10 @@ private void addTrustManagerService(OperationContext context, ModelNode ssl, Ser
         final ServiceBuilder<TrustManager[]> serviceBuilder;
         char[] keystorePassword = KeystoreAttributes.KEYSTORE_PASSWORD.resolveModelAttribute(context, ssl).asString()
                 .toCharArray();
-        String provider = KeystoreAttributes.KEYSTORE_PROVIDER.resolveModelAttribute(context, ssl).asString();
-
-        if (JKS.equals(provider) == false) {
-            ProviderTrustManagerService trustManagerService = new ProviderTrustManagerService(provider, keystorePassword);
+        final String provider = KeystoreAttributes.KEYSTORE_PROVIDER.resolveModelAttribute(context, ssl).asString();
 
+        if (!JKS.equalsIgnoreCase(provider)) {
```

Comment:
In this class, there is another occurence of "JKS.equal(provider)" . Please make it consistent.